editor/editor_property_name_processor.cpp

```diff
@@ -55,20 +77,20 @@ String EditorPropertyNameProcessor::_capitalize_name(const String &p_name) const
 	return capitalized;
 }
 
-String EditorPropertyNameProcessor::process_name(const String &p_name) const {
-	const String capitalized_string = _capitalize_name(p_name);
-	if (EDITOR_GET("interface/editor/translate_properties")) {
-		return TTRGET(capitalized_string);
-	}
-	return capitalized_string;
-}
+String EditorPropertyNameProcessor::process_name(const String &p_name, Style p_style) const {
+	switch (p_style) {
+		case STYLE_CAPITALIZED: {
+			return _capitalize_name(p_name);
+		} break;
+
+		case STYLE_LOCALIZED: {
+			return TTRGET(_capitalize_name(p_name));
+		} break;
 
-String EditorPropertyNameProcessor::make_tooltip_for_name(const String &p_name) const {
-	const String capitalized_string = _capitalize_name(p_name);
-	if (EDITOR_GET("interface/editor/translate_properties")) {
-		return capitalized_string;
+		default: {
```

Comment:
I'd use `STYLE_RAW` (and then put it first to use the same order as the enum) to make it explicit that all cases are intentionally handled. Otherwise it looks like nothing uses `STYLE_RAW`.